### ISSUE TITLE
Device Icon Colours

### ISSUE THREAD

**Issue by ChessDragon136** [TasmoAdmin/TasmoAdmin#350]
Good Day,

I would like to ask for some customization / changes in regards to the colours used for the device on / off / error / null icons. Unfortunately I suffer from some colour impairment, and the current colour just don't contrast enough for me to distinguish them. 

For me, the dark green for on looks almost identical to the black for off, I tried to replace the bulb_1_on.png but it seems to get its colour overwritten by the v=160 on the next polling. Even if you could help me change it manually I'd really appreciate it.

Kind Regards

**Comment by reloxx13** [comment]
Heya,

im sorry but currently there is no easy way to change it.

but replacing them should have worked, too. the v=160 is just to force a cache update (v=160).
if i set it to 161 it will force your browser to load the image from disk and not from cache the first time it loads.

The complicated (and untested) way is to add new on/off icons in ´tasmoadmin\resources\img\´

eg. ´mybulb_on´, ´mybulb_off´,´mybulb_none´,´mybulb_error´


Now in the devices.csv you can change the used image

´4,Ventilator,192.168.178.63,,,bulb_1,13´

bulb_1 (default) to mybulb

´4,Ventilator,192.168.178.63,,,my_bulb,13´

**Comment by Sk8wine** [comment]
Yeah, I would like the option to change the green - very hard to see the difference between green/black on certain devices.

**Comment by reloxx13** [comment]
im also open for new icons/colors to add. im not a designer, i cannot do it myself.

**Comment by Sk8wine** [comment]
I am a designer. I could help put some initial icons together? As someone who can't program, would be nice to give back to this community some how.

**Comment by reloxx13** [comment]
> 
> 
> I am a designer. I could help put some initial icons together? As someone who can't program, would be nice to give back to this community some how.

this would be really nice! 👍 😄

**Comment by Sk8wine** [comment]
If you could email me the initial image, embedded in the program, I'll try and put a few together at the weekend.

Is it was possible to give an icon drop down/selector option where you rename the devices? What are the top devices, because you could change out for say a light bulb icon etc?

**Comment by reloxx13** [comment]
i just have those added here:

https://github.com/reloxx13/TasmoAdmin/tree/master/tasmoadmin/resources/img/device_icons



> Is it was possible to give an icon drop down/selector option where you rename the devices? What are the top devices, because you could change out for say a light bulb icon etc?

yes, ill add this next.

**Comment by Sk8wine** [comment]
Amazing.

Should have known there was an image folder tree... Amazing. I'll get a few more designed.

Any idea of top smart devices - 'bulb' would be one?

**Comment by reloxx13** [comment]
bulb/light, switch, fan, sensor only, socket, blinds (up/down).

**Comment by Sk8wine** [comment]
Ah meine brude, du bist Deútsch? Mein Deútsch ist sehr slecht.

I have designed some icons this evening, as an initial concept test. Hopefully we can expand on the initial designs, if wanted/needed. Where should I send them?

They are templated off the original artwork. I have made a couple of examples of what I'd imagined, below:
 
![example1](https://user-images.githubusercontent.com/67349166/89463073-4acff480-d766-11ea-8e24-5df82266d2d0.jpg)
![example2](https://user-images.githubusercontent.com/67349166/89463079-4c012180-d766-11ea-847a-7bfc98dfce10.jpg)
